### DISCUSSION TITLE
[Feature] Ability to hide images, on the "Photos" page.

### DISCUSSION THREAD

**Discussion by Melon512** [immich-app/immich#10230] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Ability to hide images, on the "Photos" page. If this could be automated, that would be great, but just having a "hide" feature would be great. Here's a couple of example use cases:

**Example 1:**
Maybe I have have some Albums called "Wallpapers", "2FA Backup Codes", "NSFW", etc. I might not want those images on the base "Photos" page, but I still want them in their respective Albums. Being able to "Hide" these images would be a good feature to have.

**Example 2:**
Maybe I just broke up with an ex-partner. Instead of deleting those images, I could just "Hide" them. That way they're more easy to recover, as they're not actually deleted, just hidden.

I personally would think it would be good to have some kind of option, maybe a checkbox on an images "Info" section, or an Albums "Edit" page; Or, maybe when an image is hidden, it would create a new "Hidden" album, similar to what some photo apps do.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can archive files to hide them from the timeline.

**Reply by toreador34** [reply]
we need feature like this

https://docs.photoprism.app/user-guide/organize/private/
